app/src/main/java/org/p2p/wallet/striga/signup/presetpicker/SelectItemProvider.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.striga.signup.presetpicker
+
+import androidx.recyclerview.widget.RecyclerView
+
+interface SelectItemProvider {
```

Comment:
SelectItemProvider is a main abstraction in this case. 
Implement this if you want to provide your own items in the universal list - countries, tokens, just plain strings - any. This is a data source for the universal list.

Reply:
I didn't get what this is . 
Could you attach some screenshots please?